exonum-java-binding-core/src/main/java/com/exonum/binding/proxy/NativeHandle.java

```diff
@@ -53,24 +63,21 @@ public long get() {
 
   @Override
   public void close() {
-    if (isValid()) {
+    if (isValid) {
```

Comment:
I'd consider keeping the method calls as more refactoring-friendly (if we change what isValid means)